backend/Dockerfile

```diff
@@ -2,17 +2,17 @@ FROM debian:testing as builder
 
 RUN apt-get update && \
     apt-get install -y --no-install-recommends \
-        python3 python3-dev python3-pip python-is-python3 python3-venv build-essential libpq-dev libcairo2-dev libgirepository1.0-dev \
-        gir1.2-ostree-1.0 gir1.2-appstream-1.0 flatpak
+    python3 python3-dev python3-pip python-is-python3 python3-venv build-essential libpq-dev libcairo2-dev libgirepository1.0-dev \
+    gir1.2-ostree-1.0 gir1.2-appstream-1.0 flatpak
 
-RUN python -m pip install --break-system-packages pipx && \
+RUN python -m pip install --no-cache-dir --break-system-packages pipx && \
```

Comment:
Yes, this is the build itself, so likely not really worth it. But I think it might have a size impact on the layer itself. But the server likely has enough resources.

https://github.com/hadolint/hadolint/wiki/DL3042